site/docsource/Installation.adoc

```diff
@@ -16,7 +16,7 @@ is installed, run the following command:
 
 [source,sh]
 ------------------------------
-npm install --save bs-platform
+npm install --save-dev bs-platform
```

Comment:
It's a little more complicated, during the compilation.
BuckleScript also generate binary files like `.cmi` (it's similar to pre-compiled header files) in addition to `js` files. That's why BuckleScript gains much faster compilation speed and more scalability when interacting with more dependencies.
I am not sure it's great to distribute binary files (though the format it's platform-independent).
So if users are using `npm3` (or install Buckle globally), this will not add too much overhead for users, no?